src/blockdev.rs

```diff
@@ -885,11 +885,17 @@ fn split_blkid_line(line: &str) -> HashMap<String, String> {
 }
 
 fn blkid() -> Result<Vec<HashMap<String, String>>> {
+    // Run blkid with a clean cache to avoid collecting old devices which no
+    // longer exist.
+    // https://github.com/coreos/coreos-installer/pull/1288#discussion_r1312008111
+
     // Run once to gather the list of devices, and then run again with -p so
     // that we don't rely on blkid cache:
```

Comment:
```suggestion
    // Run once to gather the list of devices, which we need to specify for
    // the blkid -p below, which we use to probe the devices to not rely on
    // the blkid cache:
```

This comment might need some clarification.. we need to run the first blkid command to gather the list of devices because when specifying `blkid -p` you have to pass a list of devices.

That last part wasn't clear to me today until I ran `blkid -p` without any arguments and I got the error.

Reply:
SGTM, updated, thanks!